src/harness/fake_sas.py

```diff
@@ -0,0 +1,154 @@
+"""A fake implementation of SasInterface, based on v1.0 of the SAS-CBSD TS.
```

Comment:
Can you include the winnforum licensing headers on the .py files? You can copy from other .py files in the repository.